docs/app/pages/components/sections/date-time-picker/time-picker/time-picker.component.html

```diff
@@ -0,0 +1,122 @@
+                        </div>
+                    </div>
+                    <div class="col-md-6 col-sm-6">
+                        <div>
+                            <label for="maxMinute">max minute</label>
+                            <ux-number-picker
+                                [(value)]="maxMinute" min="0" max="59" (valueChange)="maxTime.setMinutes($event)">
+                            </ux-number-picker>
+                        </div>
+                    </div>
+                </div>
+                <div class="row uxd-customize-row">
+                    <div class="col-md-6 col-sm-6">
+                        <div>
+                            <label for="hstep">hours step</label>
+                            <ux-select [(value)]="hstep" [options]="options.hstep"></ux-select>
+                        </div>
+                    </div>
+                    <div class="col-md-6 col-sm-6">
+                        <div>
+                            <label for="mstep">minutes step</label>
+                            <ux-select [(value)]="mstep" [options]="options.mstep"></ux-select>
+                        </div>
+                    </div>
+                </div>
+                <div class="row m-t-md">
+                    <div class="col-md-12 col-sm-12">
+                        <div>
+                            <ux-checkbox [(value)]="checkModel.showMeridian" (click)="ismeridian = !ismeridian">12H / 24H</ux-checkbox>
+                        </div>
+                    </div>
+                </div>
+                <div class="row m-t-md">
+                    <div class="col-md-12 col-sm-12">
+                        <div>
+                            <ux-checkbox [(value)]="checkModel.showSeconds" (click)="showSec = !showSec">
+                                show / hide seconds
+                            </ux-checkbox>
+                        </div>
+                    </div>
+                </div>
+                <div class="row m-t-md">
+                    <div class="col-md-12 col-sm-12">
+                        <label>disable:</label>
+                        <div>
+                            <ux-checkbox [(value)]="checkModel.readonlyInput" (click)="isEnabled=!isEnabled">input</ux-checkbox>
+                        </div>
+                        <div>
+                            <ux-checkbox [(value)]="checkModel.mousewheel" (click)="mousewheel=!mousewheel">mouse wheel</ux-checkbox>
+                        </div>
+                        <div>
+                            <ux-checkbox [(value)]="checkModel.arrowkeys" (click)="arrowkeys=!arrowkeys">arrow keys</ux-checkbox>
+                        </div>
+                    </div>
+                </div>
+            </accordion-group>
+        </accordion>
+    </div>
+</div>
+
+<hr>
+
+<p>The timepicker can be created using the <code>timepicker</code> element. The <code>ngModel</code> should be set to the
```

Comment:
In terms of documentation, we really want to emphasize that this is an external component. Suggested wording would be:

> The Bootstrap `timepicker` component can be used to select a valid time. The value can be bound as a JavaScript `Date` object with `ngModel`. Additional customization options can be seen in the example above; see the Angular Bootstrap documentation for full details.